src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -0,0 +1,112 @@
+
+                        <label class="form-check-label"
+                               :for="`id-${uploadFile.index}`"
+                               v-translate="uploadFile.displayName"></label>
+
+                        <small v-if="uploadFile.resourceId" class="text-danger row">
+                        <span class="col-auto">
+                        <span v-translate="'uploadFileOverwrite'"></span>{{ lastModified(uploadFile.lastModified) }}
+                        </span>
+                        </small>
+                    </div>
+                </div>
+                <div class="mt-3">
+                    <label for="description-id" v-translate="'uploadFileDesc'"></label>
+                    <textarea v-model="uploadDescToAdr"
+                              class="form-control" rows="3"
+                              id="description-id"></textarea>
+                </div>
+            </div>
+            <template v-slot:footer>
+                <button
+                    type="button"
+                    class="btn btn-red"
+                    v-translate="'ok'"></button>
+                <button
+                    type="button"
+                    class="btn btn-white"
+                    @click.prevent="handleCancel"
+                    v-translate="'cancel'"></button>
+            </template>
+        </modal>
+    </div>
+</template>
+
+<script lang="ts">
+    import Vue from "vue";
+    import Modal from "../Modal.vue";
+    import {Dict, UploadFile} from "../../types";
+    import {BaselineState} from "../../store/baseline/baseline";
+    import {formatDateTime, mapStateProp, mapStateProps} from "../../utils";
+    import {ADRState} from "../../store/adr/adr";
+
+    interface Methods {
+        handleCancel: () => void
+        lastModified: (date: string) => string | null
+    }
+
+    interface Computed {
+        dataset: string
+        uploadFiles: Dict<UploadFile>
+    }
+
+    interface Data {
+        uploadFilesToAdr: []
+        uploadDescToAdr: string
+    }
+
+    interface Props {
+        open: boolean
+    }
+
+    export default Vue.extend<Data, Methods, Computed, Props>({
+        name: "UploadModal",
+        props: {
+            open: {
+                type: Boolean
+            }
+        },
+        data(): Data {
+            return {
+                uploadFilesToAdr: [],
+                uploadDescToAdr: ""
+            }
+        },
+        methods: {
+            handleCancel() {
+                this.$emit("close")
+            },
+            lastModified: function (date: string) {
+                return formatDateTime(date)
+            }
+        },
+        computed: {
+            ...mapStateProps<BaselineState, keyof Computed>("baseline", {
+                dataset: state => state.selectedDataset?.title
+            }),
+            uploadFiles: mapStateProp<ADRState, Dict<UploadFile>>("adr",
+                (state: ADRState) => state.uploadFiles!
```

Comment:
My PR will ensure this is populated this by triggering the relevant action on mount

Reply:
This is already being populated, mounted in download page. Emma and I discussed adding it to my ticket, sorry for not informing you.

Reply:
Sorry @mwoodbri I'd forgotten you'd picked up that ticket. Lekan and I were discussing how to modify his previous fake upload files data to conform with current master, and it seemed that might be easier to just use the state values directly.